src/metabase/models/field_values.clj

```diff
@@ -175,6 +175,12 @@
 ;;; |                                                  Utils fns                                                     |
```

Comment:
I think it'd be good if we update the docstring at the top of this to include the new `inactive` for scanning.